server/errorHandler.ts

```diff
@@ -14,7 +14,7 @@ export default function createErrorHandler(production: boolean) {
 
     return res.render('pages/error', {
       message: production ? null : error.message,
```

Comment:
I think we should remove these error message displays and just display the expected content in all environments - that would simplify this function too as we wouldn't need to pass  `production` in

More importantly though we need to log errors - I have run this locally and it looks as though this is built into the framework somewhere as I can see them on the command line OK. Let's just check we get them in AppInsights once this is merged